### ISSUE TITLE
Is there any value in running this on a 2013 to 2016 upgrade

### ISSUE THREAD

**Issue by msilinda** [MicrosoftDocs/OfficeDocs-SharePoint#277]
Please add whether there is any value in running this when performing a database attach upgrade from a 2013 farm to a 2016 farm.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5bc1ccc3-1713-6a8f-8120-28ca04a7eed8
* Version Independent ID: d0ac879e-81d4-ca80-b83d-ea1e318a9fa8
* Content: [SharePoint Migration Assessment Tool Scan Reports Roadmap](https://docs.microsoft.com/en-us/sharepointmigration/sharepoint-migration-assessment-toolscan-reports-roadmap)
* Content Source: [migration/sharepoint-migration-assessment-toolscan-reports-roadmap.md](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/blob/live/migration/sharepoint-migration-assessment-toolscan-reports-roadmap.md)
* Product: **sharepoint-server-itpro**
* GitHub Login: @JoanneHendrickson
* Microsoft Alias: **jhendr**

**Comment by AndreaBarr** [comment]
@msilinda Thank you for this feedback

I agree the advantages should be included in the article. 

I was able to get this into the backlog for the SharePoint team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated. 
 
I made a note to request the team to update this when the work is complete.

**Comment by msilinda** [comment]
I know you closed this, but can you verify whether this tool can be run on a SP 2013 farm when preparing for a database attach upgrade to a 2016 farm, as long as the person running it knows the differences between 2016 and SPO?

**Comment by AndreaBarr** [comment]
@msilinda I have gotten it over to the SharePoint team so that they can verify this. If you need an expedited answer Support can handle this as well.

**Comment by AndreaBarr** [comment]
@tseward is investigating this issue.

**Comment by tseward** [comment]
@msilinda I don't believe there is a high value. The target for SPMT is SPO, of course, where many of the rules the tool runs don't apply to on-prem SharePoint. There are also no explicit rules that capture changes from SharePoint version to version on-prem (e.g. deprecation or removal of features).
